Core/include/Acts/Utilities/Definitions.hpp

```diff
@@ -111,36 +111,88 @@ using ActsRowVectorX = Eigen::Matrix<T, 1, Eigen::Dynamic>;
 using ActsRowVectorXd = ActsRowVectorX<double>;
 using ActsRowVectorXf = ActsRowVectorX<float>;
 
-using Rotation3D = Eigen::Quaternion<double>;
-using Translation3D = Eigen::Translation<double, 3>;
-using AngleAxis3D = Eigen::AngleAxisd;
-using Transform3D = Eigen::Affine3d;
-using Vector3D = Eigen::Matrix<double, 3, 1>;
-using Vector2D = Eigen::Matrix<double, 2, 1>;
-using RotationMatrix3D = Eigen::Matrix<double, 3, 3>;
+/// @defgroup coordinates Fixed-size vector/matrix types for coordinates
```

Comment:
This is a too specific description. Could you rephrase it such that these represent `e.g.` coordinates etc.?

Reply:
Fixed.